src/email.rs

```diff
@@ -71,28 +68,9 @@ pub fn request(app: &AppConfig, params: &QueryMap) -> Value {
         .build().unwrap();
     // TODO: Add support for authentication.
     let mut mailer = SmtpTransportBuilder::new(app.smtp.address.as_str()).unwrap().build();
-    let result = mailer.send(email);
+    mailer.send(email).unwrap();
     mailer.close();
 
-    // TODO: for debugging, this part returns a JSON response with some
```

Comment:
These changes seem unrelated to the commit message, can you please put them in a separate commit?